modules/living_spaces_default/src/LivingSpacesDefaultRelationshipStorage.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\living_spaces_default;
+
+use Drupal\group\Entity\Storage\GroupRelationshipStorage;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Defines the storage handler class for relationship entities.
+ */
+class LivingSpacesDefaultRelationshipStorage extends GroupRelationshipStorage {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function save(EntityInterface $entity) {
+    if ($entity->getGroup()->get('is_default')->getString()) {
```

Comment:
This should only happen to group_membership.  
\Drupal\living_spaces_default\Access\LivingSpacesDefaultAccessCheck::access already handles access to adding content to space templates. If user is allowed to add content to space tempate, this code will remove relationship and related entity will just remain hanging

Reply:
done